platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/ConfigUtils.java

```diff
@@ -34,32 +34,39 @@
  */
 public final class ConfigUtils {
 
+    private static final String SWIRLDS_PACKAGE = "com.swirlds";
+    private static final String HEDERA_PACKAGE = "com.hedera";
+
     private ConfigUtils() {}
 
     /**
-     * Scan all classes in a classpath and register all configuration data types with a configuration builder.
+     * Scan all classes in classpath/modulepath and register all configuration data types with a configuration builder.
```

Comment:
I think it would be better, if this method still scans all packages, but `TestConfigBuilder` calls `scanAndRegisterAllConfigTypes` with specific packages by default.